superset/assets/src/explore/components/DisplayQueryButton.jsx

```diff
@@ -222,6 +233,20 @@ export default class DisplayQueryButton extends React.PureComponent {
         pullRight
         id="query"
       >
+        {this.props.slice && (
+          <>
+            <MenuItem onClick={this.openPropertiesModal}>
+              {t('Edit properties')}
+            </MenuItem>
+            <PropertiesModal
+              slice={this.props.slice}
+              show={this.state.propertiesModalOpen}
+              onHide={this.closePropertiesModal}
+              onSave={this.onSliceSave}
```

Comment:
Is this defined anywhere? Seems to be overridden in the actual component with a redux action.

Reply:
oops, yeah that shouldn't be there anymore